dashboard/test/ui/features/gdpr_dialog.feature

```diff
@@ -0,0 +1,49 @@
+@dashboard_db_access
+
+Feature: GDPR Dialog - data transfer agreement
+
+  Scenario: EU user sees the GDPR Dialog on dashboard, opt out
+    Given I am a teacher
+    Given I am on "http://studio.code.org/home?force_in_eu=1"
+    Then element ".ui-test-gdpr-dialog" is visible
+    Then I click selector ".ui-test-gdpr-dialog-logout"
+    Then check that I am on "http://code.org/"
+    Then I wait until element ".ui-test-gdpr-dialog" is not visible
+
+  Scenario: EU user sees the GDPR Dialog on pegasus, opt out
+    Given I am on "http://code.org/"
+    And I am a teacher
+    And I am on "http://code.org/teacher-dashboard?no_home_redirect=1&force_in_eu=1"
```

Comment:
I worry just a _little_ that `/teacher-dashboard?no_home_redirect=1` isn't a realistic test of this behavior, and we should actually create a section and navigate to its progress tab so we're checking this on a supported code path.

That said, we may remove support for `/teacher-dashboard?no_home_redirect=1` when we finish removing angular over the summer, and could update this test (and others) at that time, so not blocking?

Reply:
If they were really in the EU they would encounter the dialog on dashboard while going through the process of setting up a section.  I guess I could still force_in_eu on the pegasus after doing the set up in dashboard.  Does the seem more realistic? Maybe more consistent with reality in the sense that a teacher could've set up sections before GDPR, and now sees the dialog for the first time when they go to their teacher dashboard.

Reply:
Ah, that's a good point.  This is when I _really_ wish fixtures and/or direct DB access were more viable in our UI tests, so we could trivially set up our initial conditions.

Let's worry about this later. :man_shrugging: